### PR TITLE
ec2_instance: add placement_group support

### PR DESCRIPTION
**Pull request by fzwart**
##### SUMMARY
I've added ec2 placement_group support for the ec2_instance module

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
ec2_instance

### CODE REVIEW

**Review by Shaps — APPROVED** [review]

File: lib/ansible/modules/cloud/amazon/ec2_instance.py [hunk 1]
```diff
@@ -180,10 +180,14 @@
     default: {"tag:Name": "<provided-Name-attribute>", "subnet-id": "<provided-or-default subnet>"}
   instance_role:
     description:
-    - The ARN or name of an EC2-enabled instance role to be used. If a name is not provided in arn format
-      then the ListInstanceProfiles permission must also be granted.
-      U(https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListInstanceProfiles.html) If no full ARN is provided,
-      the role with a matching name will be used from the active AWS account.
+     - The ARN or name of an EC2-enabled instance role to be used. If a name is not provided in arn format
```
Comment (resmo) [inline lib/ansible/modules/cloud/amazon/ec2_instance.py]: could you check the indentation here?

Comment (fzwart) [reply lib/ansible/modules/cloud/amazon/ec2_instance.py]: done!

**Comment by Shaps** [comment]
shipit

**Comment by gundalow** [comment]
Merged into `devel` for release in Ansible 2.8, thanks!

**Pull request merged**